src/DssSpell.sol

```diff
@@ -74,190 +120,324 @@ contract DssSpellAction is DssAction {
+        // Poll: https://vote.makerdao.com/polling/QmdnSKPu#poll-detail
+        MCD_VEST_MKR_TREASURY.restrict(
+            MCD_VEST_MKR_TREASURY.create(
+                CHRONICLE_LABS,            // usr
+                2_216.4 ether,             // tot; note: ether is a keyword helper, only MKR is transferred here
+                JUL_01_2023,               // bgn
+                JUN_30_2024 - JUL_01_2023, // tau
+                0,                         // eta
+                address(0)                 // mgr
+            )
+        );
+
+        // Jetstream Auditor Wallet      | 2023-06-26 00:00:00 to 2024-12-31 23:59:59 | 1,619.93 MKR | 0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962
+        // Forum: https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306
+        MCD_VEST_MKR_TREASURY.restrict(
+            MCD_VEST_MKR_TREASURY.create(
+                JETSTREAM,                 // usr
+                1_619.93 ether,            // tot; note: ether is a keyword helper, only MKR is transferred here
+                JUN_26_2023,               // bgn
+                DEC_31_2024 - JUN_26_2023, // tau
+                0,                         // eta
+                address(0)                 // mgr
+            )
+        );
 
-        // PBG - 9.92 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
-        MKR.transfer(PBG_WALLET,              9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // ----- Ecosystem Actor Dai Transfer -----
 
-        // Flip Flop Flap Delegate - 9.92 MKR - 0x3d9751EFd857662f2B007A881e05CfD1D7833484
-        MKR.transfer(FLIPFLOPFLAP_WALLET,     9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // Jetstream - 494,001 DAI - 0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962
+        // Forum: https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306
+        DssExecLib.sendPaymentFromSurplusBuffer(JETSTREAM, 494_001);
 
-        // Bandhar - 9.92 MKR - 0xE83B6a503A94a5b764CCF00667689B3a522ABc21
-        MKR.transfer(BANDHAR_WALLET,          9.92 ether); // NOTE: 'ether' is a keyword helper, only MKR is transferred here
+        // ----- Core Unit MKR Vesting Transfer -----
 
-        // --- Add Chainlink Keeper Network Treasury Address ---
-        // Forum: https://forum.makerdao.com/t/poll-notice-keeper-network-follow-up-updates/21056
-        // Forum: https://forum.makerdao.com/t/consolidated-action-items-for-2023-06-28-executive/21187
-        // Poll: https://vote.makerdao.com/polling/QmZZJcCj#vote-breakdown
+        // DECO-001 - 125    MKR - 0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7
+        // Mip: https://mips.makerdao.com/mips/details/MIP40c3SP36#mkr-vesting
+        MKR.transfer(DECO_WALLET,    125    ether); // note: ether is a keyword helper, only MKR is transferred here
```

Comment:
Subjective, but I believe the vertical visual alignment with the next section is making this harder to read instead of easier:
```suggestion
        // DECO-001 - 125 MKR - 0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7
        // Mip: https://mips.makerdao.com/mips/details/MIP40c3SP36#mkr-vesting
        MKR.transfer(DECO_WALLET, 125 ether); // note: ether is a keyword helper, only MKR is transferred here
```